content/content-strategy/understand-content-ecosystem/show-content-ecosystem/simple-ecosystem-map.md

```diff
@@ -2,14 +2,14 @@
 layout: blocks/image_content_landscape
 image: content-strategy/ecosystem/good-ecosystem-map.jpg
 link: content-strategy/ecosystem/good-ecosystem-map.jpg
-title: Good mind map
+title: Simple ecosystem map
 reverse: false
 stackPosition: top
 imageAlt: A simple and clear content channel mind map showing just a few channels.
-caption: A good channel ecosystem has few channels and little duplication. The ecosystem is clear and simple. This diagram shows a mind map with the main website in the centre surrounded by four content channel types, websites, email newsletters, social media and apps. There are just one or two subsets connected in a clear and simple way.
-level: 2
+caption: This diagram shows a mind map with the main website in the centre surrounded by four content channel types, websites, email newsletters, apps, and social media channels – Facebook, Twitter, LinkedIn and Instagram. There are just one or two subsets connected in a clear and simple way.
+level: 3
 display: 3
```

Comment:
@annekemyers not `display:2` ? Looks like this property was left to its default by accident.